### PR TITLE
First automated test cases

### PR DESCRIPTION
**Pull request by maryfranz**
* Removed test samples from integration folder
* Added test files: login.spec.json, cart.spec.json, purchase.spec.json
* Modified cypress.json

### CODE REVIEW

File: cypress/integration/cart.spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    cy.get('button#checkout').should('be.visible');
+    // User expects to see only two added products, the removed one is not present on the cart list
+    cy.get('.inventory_item_name').should(($list) => {
+      expect($list).to.have.length(2);
+      expect($list).to.contain('Sauce Labs Fleece Jacket');
+      expect($list).to.contain('Sauce Labs Bolt T-Shirt');
+      expect($list).to.not.contain('Sauce Labs Backpack');
+    });
+  });
+});
```
Comment (robertocgongora) [inline cypress/integration/cart.spec.js]: don't forget to add a EOF new line

File: cypress/integration/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    cy.get('div.error-message-container').should('be.visible').contains('Username is required');
+  });
+
+  it('Verify login validation when no password is set', ()=> {
+    cy.get('input#user-name').type('standard_user');
+    cy.get('input#login-button').click();
+    // An error message is thrown indicating that Password is required
+    cy.get('div.error-message-container').should('be.visible').contains('Password is required');
+  });
+});
```
Comment (robertocgongora) [inline cypress/integration/login.spec.js]: same observation as above

File: cypress/integration/purchase.spec.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    cy.get('a.shopping_cart_link').click();
+    cy.get('button#checkout').should('be.visible').click();
+    // Filling some data in checkout form
+    cy.get('input#first-name').type('Mary');
+    cy.get('input#last-name').type('Rical');
+    cy.get('input#continue').click();
+    // An error message is thrown indicating that the Postal code is required
+    cy.get('div.error-message-container').should('be.visible').contains('Error: Postal Code is required');
+  });
+});
```
Comment (robertocgongora) [inline cypress/integration/purchase.spec.js]: same as above

**Review by robertocgongora — COMMENTED** [review]
Awesome work!!!!go ahead, just a couple of observations

File: cypress/integration/purchase.spec.js (hunk 3 above)
Comment (maryfranz) [reply cypress/integration/purchase.spec.js]: Thanks for the comments, I have added the new line at the EOF

**Pull request merged**